charts/fprl-dgs/values.yaml

```diff
@@ -4,15 +4,13 @@ java:
     ingressHost: "fprl-dgs-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
     environment:
         AUTH_PROVIDER_SERVICE_CLIENT_BASEURL: "http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
-        EVIDENCE_MANAGEMENT_CLIENT_API_BASEURL: "http://div-emca-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
```

Comment:
em-api is not needed. It was overdisigned divorce solution. emca was just a proxy to dm store. We can call dm-store and case doc am api directly.